addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -593,6 +593,7 @@ export class LinkPlugin extends Plugin {
             showReplaceTitleBanner: this.newlyInsertedLinks.has(linkElement),
             allowCustomStyle: this.config.allowCustomStyle,
             allowTargetBlank: this.config.allowTargetBlank,
+            allowStripDomain: this.config.allowStripDomain,
```

Comment:
Do you think it's better to make the `allowStripDomain` true by default? Cause it's an useful feature for editors out of website too :thinking:  cc @lebl-odoo  :pray: 

For that you can use `static defaultConfig`